Library/Client.ts

```diff
@@ -234,10 +234,11 @@ class Client {
      */
     public track(
         data: ContractsModule.Contracts.Data<ContractsModule.Contracts.Domain>,
-        tagOverrides?: { [key: string]: string; }) {
+        tagOverrides?: { [key: string]: string; },
+        contextObject?: any) {
```

Comment:
the point is to have this object weakly typed so we can pass `http.ClientRequest` from one auto-collection logic and `http.ServerRequest` from another. If strongly typed - what properties will we have?